src/Tools.Migration/test/Microsoft.Graph.Tools.Migration.Tests.ps1

```diff
@@ -0,0 +1,58 @@
+using module "..\Microsoft.Graph.Tools.Migration.psm1"
+Describe "New-MgMigrationPlan Command" {
```

Comment:
Let's also add test cases to cover scenarios where `# Select-MgProfile beta`/ commands are commented out.